packages/class/jsr.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@classmodel/class",
+  "version": "0.0.0",
+  "exports": {
```

Comment:
This duplicates part of the `package.json`. Can the package exports be omitted from `package.json` if we commit to JSR?

Reply:
Nope, thats how you can import the package in the app.